apps/android/app/src/main/res/values-de/strings.xml

```diff
@@ -4,73 +4,73 @@
+  <string name="trying_to_connect_to_server_to_receive_messages">Versuche die Verbindung mit dem Server aufzunehmen, der für den Empfang von Nachrichten mit diesem Kontakt genutzt wird.</string>
 
   <!-- Item Content - ChatModel.kt -->
-  <string name="deleted_description">deleted</string>
-  <string name="sending_files_not_yet_supported">sending files is not supported yet</string>
-  <string name="receiving_files_not_yet_supported">receiving files is not supported yet</string>
-  <string name="sender_you_pronoun">you</string>
-  <string name="unknown_message_format">unknown message format</string>
-  <string name="invalid_message_format">invalid message format</string>
+  <string name="deleted_description">Gelöscht</string>
+  <string name="sending_files_not_yet_supported">Das Senden von Dateien wird aktuell nicht unterstützt</string>
+  <string name="receiving_files_not_yet_supported">Der Empfang von Dateien wird aktuell nicht unterstützt</string>
+  <string name="sender_you_pronoun">Du</string>
+  <string name="unknown_message_format">Unbekanntes Nachrichtenformat</string>
+  <string name="invalid_message_format">Unzulässiges Nachrichtenformat</string>
 
   <!-- PendingContactConnection - ChatModel.kt -->
-  <string name="connection_local_display_name">connection <xliff:g id="connection ID" example="1">%1$d</xliff:g></string>
-  <string name="display_name_connection_established">connection established</string>
-  <string name="display_name_invited_to_connect">invited to connect</string>
-  <string name="display_name_connecting">connecting…</string>
-  <string name="description_you_shared_one_time_link">you shared one-time link</string>
-  <string name="description_you_shared_one_time_link_incognito">you shared one-time link incognito</string>
-  <string name="description_via_contact_address_link">via contact address link</string>
-  <string name="description_via_contact_address_link_incognito">incognito via contact address link</string>
-  <string name="description_via_one_time_link">via one-time link</string>
-  <string name="description_via_one_time_link_incognito">incognito via one-time link</string>
+  <string name="connection_local_display_name">Verbindung <xliff:g id="connection ID" example="1">%1$d</xliff:g></string>
+  <string name="display_name_connection_established">Verbindung hergestellt</string>
+  <string name="display_name_invited_to_connect">Für eine Verbindung eingeladen</string>
+  <string name="display_name_connecting">verbinde …</string>
+  <string name="description_you_shared_one_time_link">Du hast einen One-Time Link geteilt</string>
+  <string name="description_you_shared_one_time_link_incognito">Du hast einen Inkognito One-Time Link geteilt</string>
+  <string name="description_via_contact_address_link">Über einen Kontaktadressen Link</string>
+  <string name="description_via_contact_address_link_incognito">Über einen Inkognito-Kontaktadressen Link</string>
```

Comment:
this is probably incorrect, in this case not the contact address link is incognito, but the user is. So "incognito" is a mode of connection here, not the property of the link